src/TerminalSettings/TerminalSettings/MainPage.xaml

```diff
@@ -8,68 +8,150 @@
+                            <FontIcon Glyph="&#xE909;" />
+                        </muxc:NavigationViewItem.Icon>
+                        <TextBlock Tag="Powershell7_Page">PowerShell 7</TextBlock>
+                    </muxc:NavigationViewItem>
+
+                    <muxc:NavigationViewItem Tag="AddNew_Nav" >
+                        <!-- Couldn't find spaceship -->
+                        <muxc:NavigationViewItem.Icon>
+                            <FontIcon Glyph="&#xE710;" />
+                        </muxc:NavigationViewItem.Icon>
+                        <TextBlock Tag="AddNew_Page">Add new</TextBlock>
+                    </muxc:NavigationViewItem>
+                    
+                </muxc:NavigationViewItem.MenuItems>
+            </muxc:NavigationViewItem>
             
-            <NavigationViewItem Icon="Keyboard"
-                                    Tag="Keybindings_Nav">
-                <TextBlock Tag="Keybindings_Page">Keyboard</TextBlock>
-            </NavigationViewItem>
 
-            <NavigationViewItem Tag="Keybindings_Nav">
-                <NavigationViewItem.Icon>
+            <muxc:NavigationViewItem Icon="Keyboard"
+                                    Tag="Keyboard_Nav">
+                <TextBlock Tag="Keyboard_Page">Keyboard</TextBlock>
+            </muxc:NavigationViewItem>
+
+            <muxc:NavigationViewItem Tag="Mouse_Nav">
+                <muxc:NavigationViewItem.Icon>
                     <FontIcon Glyph="&#xE962;" />
-                </NavigationViewItem.Icon>
-                <TextBlock Tag="Keybindings_Page">Mouse</TextBlock>
-            </NavigationViewItem>
+                </muxc:NavigationViewItem.Icon>
+                <TextBlock Tag="Mouse_Page">Mouse</TextBlock>
+            </muxc:NavigationViewItem>
 
-            <NavigationViewItem Tag="Keybindings_Nav">
-                <NavigationViewItem.Icon>
+            <muxc:NavigationViewItem Tag="CmdPalette_Nav">
+                <muxc:NavigationViewItem.Icon>
                     <FontIcon Glyph="&#xE945;" />
-                </NavigationViewItem.Icon>
-                <TextBlock Tag="Keybindings_Page">Command Palette</TextBlock>
-            </NavigationViewItem>
-
-            <NavigationViewItem Icon="Shop"
-                                    Tag="Keybindings_Nav">
-                <TextBlock Tag="Keybindings_Page">Marketplace</TextBlock>
-            </NavigationViewItem>
-
-        </NavigationView.MenuItems>
-        <NavigationView.PaneFooter>
-            <NavigationViewItem>
-                <NavigationViewItem.Icon>
+                </muxc:NavigationViewItem.Icon>
+                <TextBlock Tag="CmdPalette_Page">Command Palette</TextBlock>
+            </muxc:NavigationViewItem>
+
+            <muxc:NavigationViewItem Icon="Shop"
+                                    Tag="Marketplace_Nav">
+                <TextBlock Tag="Marketplace_Page">Marketplace</TextBlock>
+            </muxc:NavigationViewItem>

```

Comment:
You can remove all of these. We're not gonna have the Mouse, Command Palette, and Marketplace pages yet.

Reply:
![image](https://user-images.githubusercontent.com/16733941/88351356-d9765780-cd0a-11ea-8244-9dbebab6c4e9.png)